config/ftbquests/quests/chapters/immersive_engineering.snbt

```diff
@@ -1380,5 +1386,66 @@
 				player_command: false
 			}]
 		}
+		{
+			x: 0.5d
+			y: -4.5d
+			description: [
+				"Item batcher is advanced filter block made specifically for use with conveyor belts and creating batches of items for machines."
+				""
+				"It has a lot of filtering options, few modes of output (Single - Works like normal filter, if item matches it, will be redirected to output / Batch mode - All filters needs to be satisfied for all items to be redirected to the output) and emits Redstone depending on the state of Filters."

```

Comment:
*a* few modes of output
Single - Works like *a* normal filter, if *an* item matches it, *it* will be redirected to *the* output.
Batch mode - All filters *need* to *have their criteria met* for all items to be redirected to the output

This is tough to read, having a 3-4 line parenthesis within a sentence makes you lose track of what you were reading. 

The amount of corrections in this one quest is worrying to be honest. You seem to be missing `a`, `an` and `the` the vast majority of places.